templates/server/config/environment/test.js

```diff
@@ -3,7 +3,7 @@
 
   module.exports = {
     <% if(mongoose) { %>mongo: {
-      uri: 'mongodb://localhost/<%= appNameSlug %>-test'
+      uri: 'mongodb://localhost/<%= appNameSlug %>'
```

Comment:
It's more of something you opt into on purpose I would think. This is just a generator, so you should still set it up the way that works for that project. Right now we don't have anything that necessitates a test db sep from develop, but if a project did it's an easy change.